src/main/java/de/keybird/beagle/jobs/execution/IndexJobExecution.java

```diff
@@ -79,6 +79,15 @@ public void setPageRequest(Pageable pageRequest) {
     }
 
     public void execute(JobExecutionContext<IndexJob> context) {
+        // Before we do anything, let's initialize the elastic backend
+        try {
+            context.logEntry(LogLevel.Info, "Initializing elastic pipeline for attachments");
```

Comment:
Should add more context logging to indicate if initialization is required or not and also if succeeded or not.